miner.go

```diff
@@ -57,6 +57,9 @@ func Mine(c *MinerConfig) error {
 		ExcludeHourStart: c.ExcludeHrStart,
 		ExcludeHourEnd:   c.ExcludeHrEnd,
 	})
+	if imResp.Code < 0 {
```

Comment:
nice catch! would you also add a crylog.Error() here before returning to be consistent with the other error handling in this method?

Reply:
crylog added